tests/Doctrine/Tests/DBAL/Schema/PostgreSqlListTableForeignKeysTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Doctrine\Tests\DBAL\Schema;
+
+use Doctrine\DBAL\Driver\AbstractPostgreSQLDriver;
+use Doctrine\DBAL\DriverManager;
+use PHPUnit\Framework\TestCase;
+use function extension_loaded;
+use function in_array;
+
+class PostgreSqlListTableForeignKeysTest extends TestCase
+{
+    /** @var string */
+    private $testDatabaseName = 'test_pgsql_db';
+
+    /** @var string[] */
+    private $connectionParameters = [
+        'driver'   => 'pdo_pgsql',
+        'user'     => 'postgres',
+        'password' => '',
+        'host'     => 'localhost',
+        'port'     => '5432',
+    ];
+
+    protected function setUp() : void
+    {
+        if (! extension_loaded('pdo_pgsql')) {
```

Comment:
Running the test suite with the extension loaded but no database started will cause a crash… Please look into existing tests to avoid this and hardcoding database parameters above. Maybe you should proceed as in https://github.com/doctrine/dbal/blob/master/tests/Doctrine/Tests/DBAL/Functional/Driver/PDOPgsqlConnectionTest.php